HobgobDie/Assets/Scripts/PlayerMovement.cs

```diff
@@ -0,0 +1,58 @@
+﻿using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+    public float movementScalar;
+    public float maxSpeed;
+    public float jumpScalar;
+
+    private Rigidbody2D rb;
+    private bool isGrounded = false;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        rb = this.gameObject.GetComponent<Rigidbody2D>();
+    }
+
+    // Update is called once per frame
+    void Update()
```

Comment:
This should probably be inside `FixedUpdate()`
https://stackoverflow.com/questions/34447682/what-is-the-difference-between-update-fixedupdate-in-unity

Reply:
I initially tried to put the keydown stuff in fixed update but it didn't register every keypress. I moved it to update as per https://answers.unity.com/questions/294079/unity-keydown-doesnt-always-register.html, and it worked.

Reply:
I see!
I think these two links show the ideal approaches to this:
- https://docs.unity3d.com/ScriptReference/Rigidbody-velocity.html
- https://answers.unity.com/questions/1534204/input-in-update-and-physics-in-fixed-update-good-p.html

Reply:
Thanks, that worked!